### PR TITLE
[fix bug] NullPointerException

### PR DESCRIPTION
**Pull request by shijinkui**
http://localhost:9000/api/contents/dir?type=directory&_=1433328322963

where dir not exist, occur NullPointerException

### CODE REVIEW

File: app/controllers/Application.scala [hunk 1]
```diff
@@ -388,7 +386,7 @@ object Application extends Controller {
   }
 
   def openKernel(kernelId: String, sessionId: String) = ImperativeWebsocket.using[JsValue](
-    onOpen = channel => WebSocketKernelActor.props(channel, kernelIdToCalcService(kernelId), sessionId),
+    onOpen = ch => WebSocketKernelActor.props(ch, kernelIdToCalcService(kernelId), sessionId),
```
Comment (andypetrella) [inline app/controllers/Application.scala]: nah keeping `channel` doesn't really hurt here :-D

File: modules/kernel/src/main/scala-2.10/notebook/ReplCalculator.scala [hunk 2]
```diff
@@ -218,14 +221,14 @@ class ReplCalculator(
...
-              s"""
-                 |//updating deps
-                 |jars = (${deps.mkString("List(\"", "\",\"", "\")")} ::: jars.toList).distinct.toArray
-                 |//restarting spark
-                 |reset()
-                 |jars.toList
+                s"""
+                   |//updating deps
+                   |jars = (${deps.mkString("List(\"", "\",\"", "\")")} ::: jars.toList).distinct.toArray
+                                                                         |//restarting spark
```
Comment (andypetrella) [inline modules/kernel/src/main/scala-2.10/notebook/ReplCalculator.scala]: glups :-P
hehe, the code formatter tricks us here

File: notebooks/tachyon/Simple Test1.snb [hunk 3]
```diff
@@ -0,0 +1,173 @@
+{
```
Comment (andypetrella) [inline notebooks/tachyon/Simple Test1.snb]: Do you really want this test to be kept? If o maybe a bit of doc and renaming would be cool

**Comment by andypetrella** [comment]
Nice catch! Thanks a lot

Besides the small comments, it's almost ready to go!

File: notebooks/tachyon/Simple Test1.snb (hunk 3 above)
Comment (shijinkui) [inline notebooks/tachyon/Simple Test1.snb]: saved snb when local debug.  there are no problem when running.
add notebooks to .ignore. when need change notebooks' snb file, close the ignore first. OK?

File: app/controllers/Application.scala (hunk 1 above)
Comment (shijinkui) [inline app/controllers/Application.scala]: want keeping max 100 words one line :(

**Comment by andypetrella** [comment]
Cool thanks @shijinkui !

**Pull request merged**